File: sdk/src/main/java/com/cnblogs/sdk/exception/CnblogsIOException.java
package com.cnblogs.sdk.exception;

import java.io.IOException;

/**
 * IO异常
 * @author RAE
 * @date 2022/02/19
 * Copyright (c) https://github.com/raedev All rights reserved.
 */
@SuppressWarnings("AlibabaClassNamingShouldBeCamel")
public class CnblogsIOException extends IOException {

    public CnblogsIOException() {
    }

    public CnblogsIOException(String message) {
        super(message);
    }

    public CnblogsIOException(String message, Throwable cause) {
        super(message, cause);
    }

    public CnblogsIOException(Throwable cause) {
        super(cause);
    }
}

File: sdk/src/main/kotlin/com/cnblogs/sdk/exception/CnblogsIOException.kt
package com.cnblogs.sdk.exception

import java.io.IOException

/**
 * IO异常
 * @author RAE
 * @date 2022/10/04
 * @copyright Copyright (c) https://github.com/raedev All rights reserved.
 */
open class CnblogsIOException : IOException {
    constructor(message: String) : super(message)
    constructor(message: String, cause: Throwable) : super(message, cause)
}